portfolio/src/main/webapp/script-comments.js

```diff
@@ -0,0 +1,277 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Creates the pagination bar such that the active page number is the previously active page number, unless it no longer exists. 
+ * @param {number} numPages The total number of pages. 
+ * @param {number} activePageNumber The previously active page number.
+ * @return {number} The currently active page number.
+ */
+function createPaginationBar(numPages, activePageNumber) {
+  // Set the currently active page number to 1 if the previously active page number no longer exists.
+  if (activePageNumber > numPages) {
+    activePageNumber = 1;
+  }
+  
+  pageListElement = document.getElementById('page-list');
+  pageListElement.innerHTML = '';
+
+  pageListElement.appendChild(createPageElement('Previous'));
+  for (var i = 1; i <= numPages; i++) {
+    if (i == activePageNumber) {
+      pageListElement.appendChild(createPageElement(i, true));
+    } else {
+      pageListElement.appendChild(createPageElement(i, false));
+    }
```

Comment:
You could make this a lot easier to read:

```suggestion
    pageListElement.appendChild(createPageElement(i, i == activePageNumber));
```

Reply:
Sounds good!